/*
 * This file was automatically generated by EvoSuite
 * Mon May 02 08:46:55 GMT 2022
 */


import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true) 
public class Solution_288_ESTest extends Solution_288_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      Solution_288 solution_288_0 = new Solution_288();
      String string0 = solution_288_0.convert("V^", 2);
      assertEquals("V^", string0);
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      Solution_288 solution_288_0 = new Solution_288();
      // Undeclared exception!
      try { 
        solution_288_0.convert((String) null, 418);
        fail("Expecting exception: NullPointerException");
      
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("Solution_288", e);
      }
  }

  @Test(timeout = 4000)
  public void test2()  throws Throwable  {
      Solution_288 solution_288_0 = new Solution_288();
      // Undeclared exception!
      try { 
        solution_288_0.convert("", (-4110));
        fail("Expecting exception: NegativeArraySizeException");
      
      } catch(NegativeArraySizeException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("Solution_288", e);
      }
  }

  @Test(timeout = 4000)
  public void test3()  throws Throwable  {
      Solution_288 solution_288_0 = new Solution_288();
      // Undeclared exception!
      try { 
        solution_288_0.convert("ug?l4'%9>9H}iWQ8/", 0);
        fail("Expecting exception: ArrayIndexOutOfBoundsException");
      
      } catch(ArrayIndexOutOfBoundsException e) {
         //
         // 0
         //
         verifyException("Solution_288", e);
      }
  }

  @Test(timeout = 4000)
  public void test4()  throws Throwable  {
      Solution_288 solution_288_0 = new Solution_288();
      String string0 = solution_288_0.convert("'piy:f[-=3W#m", 8);
      assertEquals("'pimy#:Wf3[=-", string0);
  }

  @Test(timeout = 4000)
  public void test5()  throws Throwable  {
      Solution_288 solution_288_0 = new Solution_288();
      String string0 = solution_288_0.convert("", 8);
      assertEquals("", string0);
  }

  @Test(timeout = 4000)
  public void test6()  throws Throwable  {
      Solution_288 solution_288_0 = new Solution_288();
      String string0 = solution_288_0.convert(" 6A#euEZ&ovG}[o`", 1);
      assertEquals(" 6A#euEZ&ovG}[o`", string0);
  }
}
